internal/operators/authorino/authorino_config.go

```diff
@@ -0,0 +1,4 @@
+package authorino
+
+type Config struct {
+}
```

Comment:
why do we need this?

Reply:
We don't need it. I added it to be consistent with other operators, in case we want to add something in the future. I will remove it.

Reply:
Done.